docs/metal_device_create.md

```diff
@@ -37,7 +37,7 @@ metal device create -p <project_id> (-m <metro> | -f <facility>) -P <plan> -H <h
   -p, --project-id string                The project's UUID. This flag is required, unless specified in the config created by metal init or set as METAL_PROJECT_ID environment variable.
   -S, --public-ipv4-subnet-size int      Size of the public IPv4 subnet.
   -s, --spot-instance                    Provisions the device as a spot instance.
-      --spot-price-max float             Sets the maximum spot market price for the device: --spot-price-max=1.2
+      --spot-price-max float32           Sets the maximum spot market price for the device: --spot-price-max=1.2
```

Comment:
This should stay as `float`; the bit size is an implementation detail and doesn't really have an impact on the end user

Reply:
This is user-facing documentation, not code, so the specific type implementation doesn't have an impact on how a user uses the CLI.  The user needs to know that they should specify a floating point number.  They have no way of typing the argument as a float32 vs a float64.